### PR TITLE
Various typo and wording tweaks

### PR DESCRIPTION
**Pull request by deoren**
I'm new to making contributions via GitHub, so please let me know if I'm going about this correctly.

Thanks.

### CODE REVIEW

**Comment by deoren** [comment]
I actually meant to include the changes to the global object documentation as a separate pull request, but apparently I forgot to branch and make those changes in a separate branch. Now it is showing as an addition to the first PR instead of a separate item.

If I need to I can blow away my fork and redo the work.

**Comment by rgerhards** [comment]
Thanks for the PR. I think multiple commits are fine, actually it's along the same theme (multiple PRs would also be fine for me, just for the records ;-)).

**Comment by deoren** [comment]
Thanks @rgerhards . Going forward I'll make sure to create separate branches for changes.

File: source/rainerscript/global.rst [hunk 1]
```diff
@@ -171,6 +171,20 @@ The following parameters can be set:
   syslogtag. In those cases, the programname is truncated at the
   first slash. If this setting is changed to "on", slashes are
   permitted and will not terminate programname parsing.
+  
+  - **parser.permitSlashInProgramName** [on/off] available in 8.25.0+
```
Comment (deoren) [inline source/rainerscript/global.rst]: Note to self: Submit a new PR that fixes the spacing here.

**Pull request merged**